pkg/runctl/error.go

```diff
@@ -0,0 +1,51 @@
+package runctl
+
+import (
+	"errors"
+)
+
+// Error is a runManager error
+type Error struct {
```

Comment:
Is it necessary to export this type? If not, please make it it private.

Reply:
Changed